src/families/osmosis/js-buildTransaction.ts

```diff
@@ -0,0 +1,96 @@
+import { Account } from "../../types";
+import { Transaction } from "./types";
+import { SignMode } from "cosmjs-types/cosmos/tx/signing/v1beta1/signing";
+import { cosmos } from "@keplr-wallet/cosmos";
```

Comment:
Was this new package mandatory? No way to build payloads using the same utils as the current cosmos integration?